carbon-design-system/carbon-for-ibm-dotcom
# fix(cloud-mega-menu): adding missing hover bg color

### Related Ticket(s)

GH Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7180
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2207

### Description

- This PR adds the missing hover state background color of `$hover-ui`.

### Testing Instruction

- Visit the `Cloud Masthead` - `/?path=/story/components-masthead--default` and `/?path=/story/components-cloud-masthead--default`
- Ensure that the hover state is using the `$hover-ui` token.

### Expected Display

![Screen Shot 2021-09-21 at 1 09 55 PM](https://user-images.githubusercontent.com/1815714/134216515-e63ca703-7e33-485c-a6aa-5056fda42380.png)
